packages/user/src/components/SignupWrapper.tsx

```diff
@@ -1,29 +1,38 @@
 import { useTranslation } from "@dzangolab/react-i18n";
 import React, { useState } from "react";
+import { Link } from "react-router-dom";
 import { toast } from "react-toastify";
 
 import SignupForm from "./SignupForm";
-import { useUser } from "../hooks";
+import { useConfig, useUser } from "../hooks";
 import signup from "../supertokens/signup";
 
 import type { LoginCredentials, SignInUpPromise } from "../types";
 
+// eslint-disable-next-line import/no-unresolved
+import { ROUTES } from "@/constants";
```

Comment:
Could we remove this eslint-disable? You can use the "../" for import if needed. Same comment for LoginWrapper as well.